cfg/dist.js

```diff
@@ -14,6 +14,21 @@ const distConfig = merge( {}, baseConfig, {
 
     devtool   : 'source-map',
     externals : {
+        'lodash' : {
+            'commonjs'  : 'lodash',
+            'commonjs2' : 'lodash',
+            'window'    : '_'
+        },
+        'moment' : {
+            'commonjs'  : 'moment',
+            'commonjs2' : 'moment',
+            'window'    : 'Moment'
```

Comment:
According to the [Moment.js docs ](https://momentjs.com/docs/) the global/window variable is `'moment'` (with a small _m_) so let’s update this line:
```suggestion
            'window'    : 'moment'
```